### PR TITLE
Adds Adaptor and Aggregator with 5 new datafeeds

### PR DESCRIPTION
**Pull request by nick-bisonai**

### CODE REVIEW

File: adapter/krw-usd.adapter.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      }
+    },
+    {
+      "name": "Daum-KRW-USD",
+      "definition": {
+        "url": "https://finance.daum.net/api/exchanges/FRX.KRWUSD",
+        "headers": {
+          "Content-Type": "application/json",
+          "referer": "https://finance.daum.net/exchanges/FRX.KRWUSD",
+          "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
```
Comment (martinkersner) [inline adapter/krw-usd.adapter.json:62]: @nick-bisonai Do we have to specify the user agent to be able to access the data? 🤔

**Review by martinkersner — APPROVED** [review]
LGTM! I am just curious about the `referer` and `User-Agent` headers. Is it necessary? And I am not sure currently use the `headers` object when making requests. Please take a look into it 🙏

File: adapter/krw-usd.adapter.json (hunk 1 above)
Comment (nick-bisonai) [reply adapter/krw-usd.adapter.json:62]: yes, request is blocked without `User-Agenet` and `referer` headers :(

**Comment by bayram98** [comment]
@nick-bisonai I think you also need to update `README` and `HISTORY` to see your recent additions on `orakl-config` web page.

**Comment by nick-bisonai** [comment]
> @nick-bisonai I think you also need to update `README` and `HISTORY` to see your recent additions on `orakl-config` web page.

it has been added ;)

**Pull request merged**